orderer/consensus/follower/follower_chain.go

```diff
@@ -199,31 +235,69 @@ func (c *Chain) StatusReport() (types.ClusterRelation, types.Status) {
 	return clusterRelation, status
 }
 
+func (c *Chain) IsRunning() bool {
+	c.mutex.Lock()
+	defer c.mutex.Unlock()
+
+	if c.started {
```

Comment:
under what circumstance is a follower chain re-started after shutdown?

Reply:
A follower chain is started only once and stopped only once. Subsequent starts have no effect. I do not expect `start` to be called more than once in correct code. However,  I am protecting against it nevertheless. 

The `IsRunning` method is used only for testing so far, and checks whether the internal go-routine is running.

Note that when a follower chain shuts down "internally", it can happen for two reasons: either the follower finished on -boarding (reached join-block), or it is following the cluster (after the join block), and discovers it was added to the cluster by a config block (see next PR). In either case it then calls the "CreateChain" callback, (function pointer to the registrar's create chain), which will create either a follower.Chain or an etcdraft.Chain.

- The first scenario is when the follower chain is doing on boarding and then continues to follow the cluster;
- The second scenario is when the follower chain is either: (1) doing on boarding and then mutates into an etcdraft chain, or (2) following the cluster and then add to the consenters set.